README.md

```diff
@@ -46,6 +47,14 @@
 * [Discord Bot API for processes](https://github.com/kinode-dao/discord_api)
 * [LLM API](https://github.com/kinode-dao/llm), for Groq, OpenAi, as well as llama.cpp.
 
+### Useful Regexes
+* full process addresses: `^([a-z0-9-_.]+)@([a-z0-9\-_]+):([a-z0-9-_]+):([a-z0-9-_.]+)#([a-z]+)$`
```

Comment:
I think there is a typo here. It doesn't work as expected for, e.g.,
```
foo.os@bar:baz:template.os
```
but does work for
```
foo.os@bar:baz:template#os
```

Typo?

Reply:
ah yeah, i must've typoed it. i'm also working on a thing that involves a custom address format based on the kinode address format, that is probably why